apps/src/sites/studio/pages/blocks/edit.js

```diff
@@ -42,6 +44,7 @@ function initializeEditPage(defaultSprites) {
     typeHints: true,
     isBlockEditMode: true,
   });
+  Blockly.fieldRegistry.register('field_dance_ai', CdoFieldDanceAi);
```

Comment:
This (and the change below) are just hacks to get the block edit page to work for the Dance AI block.